src/core/psxmem.cc

```diff
@@ -92,35 +92,43 @@ int PCSX::Memory::init() {
     m_writeLUT = (uint8_t **)calloc(0x10000, sizeof(void *));
 
     m_psxM = (uint8_t *)calloc(0x00800000, 1);
-    m_psxP = (uint8_t *)calloc(0x00010000, 1);
+    m_psxP = (uint8_t *)calloc(0x00080000, 1);
```

Comment:
It's probably best to start doing away with the old names for the memory ranges. Nobody is going to read `m_psxP` and think "Oh yes this is PIO memory". Something like `m_pio` is just as short and much more clear.